test/conformance/cts_exe.py

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env python3
```

Comment:
For now it's fine, but eventually, I'd like to see this script somehow merged with #810. They fulfill a similar role...

Reply:
Sure, we can plan such changes in the future.